pkg/kubelet/apis/well_known_labels.go

```diff
@@ -18,13 +18,14 @@ package apis
 
 const (
 	LabelHostname          = "kubernetes.io/hostname"
-	LabelZoneFailureDomain = "failure-domain.beta.kubernetes.io/zone"
-	LabelZoneRegion        = "failure-domain.beta.kubernetes.io/region"
+	LabelZoneFailureDomain = "failure-domain.kubernetes.io/zone"
+	LabelZoneRegion        = "failure-domain.kubernetes.io/region"
```

Comment:
These are baked into the default scheduler startup options (see DefaultFailureDomains and the kube-scheduler failure-domains flag). I think changing this can break scheduling to version skewed kubelets, at least.